newIDE/app/src/ProjectManager/index.js

```diff
@@ -1132,6 +1144,15 @@ export default class ProjectManager extends React.Component<Props, State> {
             onChangeSubscription={this.props.onChangeSubscription}
           />
         )}
+        {this.state.extensionsPropertiesDialogOpen && (
```

Comment:
As explained, let's rather use the component in the `ProjectPropertiesDialog` :)